### ISSUE TITLE
AccessTimeObjectHolder uses non static field updater

### ISSUE THREAD

**Issue by max-neverov** [trivago/triava#3]
useCountAFU field updater is not static which means it will be created with each holder instance.
My proposal is to make it private static final so that we can reduce memory consumption.
Unfortunately, I don't have rights to create pull request.

**Comment by christian-esken** [comment]
You are right. The fix is here: https://github.com/trivago/triava/commit/56279dd8faf27999cfb254e2c3d180d1cd4af683

For bugfixes, the triava project follows the fork-and-merge style. For future contributions see https://github.com/trivago/triava/commit/3a7cdb95599fa863d6e80a0e1917b4c6e0364b98 for an example.

For this fix, I add you as  max-neverov to CHANGES.md

**Comment by christian-esken** [comment]
BTW. Incidentally I fixed this last week, when I found the following in a Thread dump:

```
pool-4-1" #73 daemon prio=5 os_prio=0 tid=0x00007fedc402a800 nid=0x1c34 runnable [0x00007fee803f4000]
   java.lang.Thread.State: RUNNABLE
        at java.lang.String.intern(Native Method)
        at java.lang.Class.searchFields(Class.java:2956)
        at java.lang.Class.getDeclaredField(Class.java:2068)
        at java.util.concurrent.atomic.AtomicIntegerFieldUpdater$AtomicIntegerFieldUpdaterImpl$1.run(AtomicIntegerFieldUpdater.java:390)
        at java.util.concurrent.atomic.AtomicIntegerFieldUpdater$AtomicIntegerFieldUpdaterImpl$1.run(AtomicIntegerFieldUpdater.java:388)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.util.concurrent.atomic.AtomicIntegerFieldUpdater$AtomicIntegerFieldUpdaterImpl.<init>(AtomicIntegerFieldUpdater.java:387)
        at java.util.concurrent.atomic.AtomicIntegerFieldUpdater.newUpdater(AtomicIntegerFieldUpdater.java:88)
        at com.trivago.triava.tcache.AccessTimeObjectHolder.<init>(AccessTimeObjectHolder.java:42)
        at com.trivago.triava.tcache.Cache.putToMapI(Cache.java:655)
        at com.trivago.triava.tcache.Cache.putToMap(Cache.java:550)
        at com.trivago.triava.tcache.Cache.put(Cache.java:427)
```

**Comment by christian-esken** [comment]
Thanks for the contribution. I will soon create a v1.0.4 release on Maven Central.

**Comment by max-neverov** [comment]
It would be great to see Maxim Neverov in number of trivago's contributors :)

**Comment by christian-esken** [comment]
> It would be great to see Maxim Neverov in number of trivago's contributors :)

I am open for that. To get in the list of contributors (which is fair) a small edit is enough, e.g. fixing your name spelling in CHANGES.md. Or do you have a different idea? Let me know your opinion.

**Comment by max-neverov** [comment]
Christian,
nope, I have no objections:)

**Comment by christian-esken** [comment]
OK. So do you want to fork-and-merge (pull request) then? If I would edit the file, your name will be in the file, but obviously not in the Github contributors. I prefer that you do the  fork-and-merge (pull request), but I am open for both options. Let me know what you want:
- You fork,  edit the CHANGES.md file and do a merge via pull request
- I edit the CHANGES.md file

**Comment by christian-esken** [comment]
Released v1.0.4
http://search.maven.org/#artifactdetails%7Ccom.trivago%7Ctriava%7C1.0.4%7Cjar
